packages/camera/camera_platform_interface/test/camera_platform_interface_test.dart

```diff
@@ -163,7 +163,33 @@ void main() {
             lensDirection: CameraLensDirection.back,
             sensorOrientation: 0,
           ),
-          ResolutionPreset.high,
+          ResolutionPreset.low,
+        ),
+        throwsUnimplementedError,
+      );
+    });
+
+    test(
+        'Default implementation of createCameraWithSettings() should throw unimplemented error',
```

Comment:
The test should be that it delegates to `createCamera` (via a partial test implementation).

Reply:
> Now default implementation of `createCameraWithSettings` calls `createCamera` in `camera_platform.dart`. 

Yes, and per my previous comment that needs to be tested.

> Need I still create partial implementation in `ExtendsCameraPlatform`?

I'm not clear on how you would test that `createCameraWithSettings`'s default implementation does a correct delegation (including params) to `createCamera` without a partial implementation, but if you have another approach in mind I'm happy to review it.

Reply:
partially implemented and tested